CMakeLists.txt

```diff
@@ -416,7 +416,7 @@ include(GNUInstallDirs)
 
 
 set(HIPSYCL_INSTALL_CMAKE_DIR
-  "lib/cmake/hipSYCL" CACHE PATH "Install path for legacy CMake config files")
+  "lib${LIB_SUFFIX}/cmake/hipSYCL" CACHE PATH "Install path for legacy CMake config files")
```

Comment:
I wonder, if we generalize this, do you think there might be use cases for having a lib-directory that starts without `lib`?

Reply:
as fedora packager I see often this pattern into cmake file `lib${LIB_SUFFIX}` 
on mac os it is `/usr/local/lib` where `/usr/local` is the prefix
on linux base redhat it is `lib${LIB_SUFFIX}` depnding if it is a 23 library or a 64 
on linuy debian based it is `lib`